src/App.tsx

```diff
@@ -14,28 +15,33 @@ function App() {
 
   return (
     <div className="App">
-      <header className="App-header">
-        <h1>Welcome to Heartbeat 🏥</h1>
-        <div
-          style={{
-            border: "1px solid white",
-            borderRadius: "8px",
-            padding: "20px",
-            marginBottom: "20px",
-          }}
-        >
-          <h2>Please load the patients using the button below or search</h2>
-          <PatientsLoader
-            loadPatients={patientsApi.All}
-            onLoaded={updatePatients}
-          />
-          <PatientsSearch
-            loadPatients={patientsApi.Search}
-            onResults={updatePatients}
-          />
-          {patients.length > 0 && displayPatients(patients)}
-        </div>
-      </header>
+      <ToastProvider>
+        <header className="App-header">
+          <h1>Welcome to Heartbeat 🏥</h1>
+          <div
+            style={{
+              border: "1px solid white",
+              borderRadius: "8px",
+              padding: "20px",
+              marginBottom: "20px",
+              display: "flex",
+              flexDirection: "column",
+              alignItems: "center",
+            }}
+          >
+            <h2>Please load the patients using the button below or search</h2>
+            <PatientsLoader
+              loadPatients={patientsApi.All}
```

Comment:
Is it possible to refactor this to avoid passing state up and down between the child and parent components? This might require editing the existing components. Happy to discuss this in the next interview!

Reply:
I added another provider and refactored the structure so we are using context api again to access state and simplify the components to prevent the confusing component boundary.